actions/docker-push/action.yaml

```diff
@@ -0,0 +1,53 @@
+name: "Push Docker image"
+description: "Push Docker image to any host"
+
+inputs:
+  docker-registry:
+    description: "Host where the image should be pushed to."
+    required: false
+    default: ""
+  image-namespace:
+    description: "Namespace of Docker image."
+    required: false
+    default: ""
+  image-name:
+    description: "Name of Docker image."
+    required: false
+    default: "${{ github.event.repository.name }}"
+  image-tag:
+    description: "Tag of Docker image."
+    required: false
+    default: "pipeline-${{ github.run_id }}-git-${GITHUB_SHA::8}"
+  working-directory:
+    description: "Working directory for your Docker artifacts."
+    required: false
+    default: "."
+runs:
+  using: "composite"
+  steps:
+    # setup-buildx action will create and boot a builder using by default the docker-container driver.
+    # This is not required but recommended using it to be able to build multi-platform images, export cache, etc.
+    - name: Set up Docker Buildx
```

Comment:
@yannick-roeder I don't know if that is needed. I just copied it. I would think that it is never necessary for docker load/push and only for build

Reply:
It's not required, but recommended for better docker functionality (as the comment states). I would just leave it, as it never hurt to have it there